src/main/features/claim/views/start.njk

```diff
@@ -10,67 +10,76 @@
+          {claimFeeUri: 'https://www.gov.uk/make-court-claim-for-money/court-fees' }) |safe }}
         </li>
-        <li>{{ t('Around 30 minutes to make your claim.') }}</li>
       </ul>
 
-      <p class="panel panel-border-wide">{{ t('You can save your claim and return to it at any time.') }}</p>
-
       <div class="clearfix">
         <a class="button button-start" href="{{ startLink }}" role="button">{{ t('Start now') }}</a>
       </div>
-
       <br>
+      <h2 class="heading-medium">{{ t('Check you can use this service') }}</h2>
+      <p>{{ t('To use this service, you must be:') }}</p>
 
-      <h2 class="heading-medium">{{ t('Other ways to claim') }}</h2>
-
-      <p>{{ t('You can print and fill in a') }} <a target="_blank"
-          href="http://hmctsformfinder.justice.gov.uk/HMCTS/GetForm.do?court_forms_id=338">{{ t('paper claim
-                form.') }}</a> {{ t('Send it and a cheque for your claim fee to the the address on the form.') }}</p>
-
-      {{ supportWidget('Help to use this service') }}
+      <ul class="list list-bullet">
+        <li>{{ t('claiming for £10,000 or less, including interest') }}</li>
+        <li>{{ t('18 or older (you must also be claiming against someone who‘s 18 or older)') }}</li>
+        <li>{{ t('living in England or Wales (and claiming against a person or organisation with an address in England or Wales)') }}</li>
+      </ul>
 
-      <h2 class="heading-medium">{{ t('Legal advice') }}</h2>
-      <p>{{ t('You might be able to get free legal advice from organisations like <a target="_blank" href="%s">Citizens Advice</a>'
-        + ' and <a target="_blank" href="%s">Advicenow</a>.',
-        { postProcess: 'sprintf', sprintf: [
-          'https://www.citizensadvice.org.uk/about-us/how-we-provide-advice/advice/',
-          'http://www.advicenow.org.uk/'
-          ] }) | safe }}
-      </p>
-      <p>{{ t('You don’t have to use a lawyer to make a money claim but some people choose to, for example, if there’s'
-        + ' a lot of money involved.') }}
-      </p>
+      <h2 class="heading-medium">{{ t('Claim by paper') }}</h2>
+      <p>{{ t('You can also print and complete a <a href="{{  paperFormUri }}">paper claim form</a>.',
+        { paperFormUri: 'http://hmctsformfinder.justice.gov.uk/HMCTS/GetForm.do?court_forms_id=338' }) | safe }}</p>
+      <p>{{ t('Send the form and a cheque for your <a href="{{ claimFeeUri }}">claim fee</a> to:',
+        { claimFeeUri: 'https://www.gov.uk/make-court-claim-for-money/court-fees'}) | safe }}</p>
+      <ul class="list">
+        <li>County Court Money Claims Centre</li>
+        <li> PO Box 527</li>
+        <li>Salford</li>
+        <li>Greater Manchester</li>
+        <li>M5 0BY</li>
+        <li>DX: 702634 Salford 5</li>
+      </ul>
 
+      <h2 class="heading-medium">{{ t('Help to use this service') }}</h2>
+      <ul class="list">
+      <li>{{ t('Telephone: 0300 123 7050 ') }}</li>
```

Comment:
Please remove whitespace after last digit. That would cause nasty surprises when we added welsh language. The same in line below please.

Reply:
done